flocker_bb/builders/flocker.py

```diff
@@ -661,6 +661,19 @@ def getBuilders(slavenames):
                 nextSlave=idleSlave,
                 ))
 
+    # Storage backend builders
+    builders.extend([
+        BuilderConfig(
+            name='flocker-storage-backend-cinder',
+            slavenames=["rackspace-blockdevice"],
```

Comment:
This wants to be `slavenames['rackspace-blockdevice']` probably.

For the actual name. It should probably have both the provider and the operating system (`rackspace/centos-7`?). I want to say that there should be some sort of tag-matching going on; but that can be added later if things get involved enough to need it.